### PR TITLE
Fix failing tests and add validation_results accessibility

### PR DESCRIPTION
**Pull request by mootrichard**
There were failing tests (not sure why they had been passing previously) that I have fixed so that this build would be passing.

I also expanded the scope of fixing those tests to include extending the Address object to correctly map validation results. I also added some tests for those additions to make sure its working moving forward.

### CODE REVIEW

**Comment by mootrichard** [comment]
This also fixes #23

File: src/main/java/com/shippo/model/Address.java [hunk 1]
```diff
@@ -33,6 +34,18 @@
...
     Object metadata;
     Object messages;
+	ValidationResults validation_results;
+
+    public class ValidationResults {
+    	boolean is_valid;
+    	List<ValidationMessages> messages;
+	}
+
+	public class ValidationMessages {
```
Comment (vinhhv) [inline src/main/java/com/shippo/model/Address.java]: Can you add a test where you parse this information out from the response and verify there's a message with a `source`, `code`, and `text`?

**Comment by mootrichard** [comment]
Should be fixed now. I had to re-run the build because the Batch failed due to timing out

**Comment by vinhhv** [comment]
@mootrichard LGTM!

**Pull request merged**